ui/hooks/useAccountTotalFiatBalance.js

```diff
@@ -54,10 +54,11 @@ export const useAccountTotalFiatBalance = (
   const primaryTokenImage = useSelector(getNativeCurrencyImage);
   const nativeCurrency = useSelector(getNativeCurrency);
 
-  const { loading, tokensWithBalances } = useTokenTracker({
+  const loading = false;
```

Comment:
We don't have an easy way to replicate the loading flag.  Both because the token balances controller doesn't currently set any state indicating an update is in progress.  And because in the multichain world, it may not be a single request we're waiting on.  Each network will update and return separately.  We don't have UI to indicate that specific balances are loading.

Reply:
Reading balances from state will have some UI benefits though.  Since we don't have to wait for requests to return, we can always immediately show what we have in state, without an initial 'loading' flash.  The values may be stale though, so we may want some indicators.